99.md

```diff
@@ -0,0 +1,170 @@
+-->
+
+```json
+{
+  "content": <outcome description>,
+  "tags": [
+    ["e", <id of predictable event>],
+    ["zap", <id of predictable event>+<outcome index>@SERVICE_URL, "lud16"]
+    ["d", <id of predictable event>+<outcome index>]
+    ["odds", <odds>, <odds format>]
+  ],
+  "kind": 301099,
+  "id": <outcome event id>,
+  "pubkey": <pubkey of bookmaker>,
+  "created_at": <timestamp>
+  ...
+}
+```
+
+These events include `zap` tags to link zapping the event with buying a share of that specific outcome with the current given odds.
+
+<!-- how to sell shares using NIP57 ??? -->
+
+Outcomes are specified by using plus addresses and zero-based indexing the `outcome` tags in the referred predictable event. Participants must validate that the plus address local part references an outcome with the same description as in `content` to prevent ambiguity.
+
+<!-- plus addresses are not supported by LUD16 ... yet? -->
+
+The `odds` tag specifies the current odds for that outcome. The events are parametrized replaceable events with the `d` tag set to the plus address local part to replace the event with updated `<odds>` as soon as possible after each additional stake. `<odds>` are in the format specified by `<odds format>` which can take the values "fractional", "decimal" or "moneyline" (see Appendix B).
+
+For example:
+
+```json
+{
+  "content": "Yes",
+  "tags": [
+    ["e", <id of predictable event>],
+    ["zap", <id of predictable event>+0@SERVICE_URL, "lud16"]
+    ["d", <id of predictable event>+0]
+    ["odds", "+148", "money line"]
+  ],
+  "kind": 301099,
+  "created_at": 1683507135
+  ...
+}
+```
+
+To calculate `<odds>`, the probability $p(x)$ the market has chosen for the outcome $x$ is required. This is done by dividing the amount of stake on a specific outcome by the total amount of stake on the event: $p(x) = \frac{stake(x)}{\sum_i stake(i)}$
+To make sure that there is always some stake to be won, a betting exchange can provide liquidity by betting on every outcome (like any other participant could).
+
+<!--
+I thought this would solve some bootstrapping issues if the betting exchange just bets on every outcome.
+So participants know there is always something to win, regardless of participants order.
+But not sure if this is a good approach??
+-->
+
+### Buying shares
+
+Participants which want to bet on the outcome of an event zap the events created by the betting exchange to buy shares.
+
+This means they create a Zap Request event request as specified in NIP-57 Appendix A with an `a` tag set (NIP-33 event coordinate).
+
+### Selling shares
+
+TBD
+
+### Payouts
+
+TBD
+
+## Reference
+
+### Appendix A
+
+<!--
+I think this video is a really good explanation of what I am trying to achieve with this NIP.
+Is this "good style" to include links to videos instead of writing it out? -->
+
+[![](https://img.youtube.com/vi/xA27x7GRMZQ/0.jpg)](https://www.youtube.com/watch?v=xA27x7GRMZQ)
```

Comment:
Add your comments regarding this to this thread

Reply:
Oh, looking at https://github.com/ekzyis/nips/blob/nip-prediction-markets/99.md, this doesn't seem to render the image on Github so one can't click on anything. So maybe just add the link without any image?